src/utils/traffic-mesh.js

```diff
@@ -84,7 +84,7 @@ const runProcess = async ({ log, args }) => {
   await installTrafficMesh({ log })
 
   const execPath = path.join(getBinPath(), EXEC_NAME)
-  execa(execPath, args, { stdio: 'inherit' })
+  await execa(execPath, args, { stdio: 'inherit' })
```

Comment:
This actually breaks `netlify dev --trafficMesh` since it waits for the process to exit.
On a side node, `netlify dev --trafficMesh` is already broken since `runProcess` should return the sub process.